README.md

```diff
@@ -12,6 +12,23 @@ const dataset = new Dataset(points);
 const scatterGL = new ScatterGL(containerElement, dataset, params);
 ```
 
+## Installation
+
+##### with yarn / npm
+
+```bash
+yarn install scatter-gl
+```
+
+##### via cdn
+
+```html
+<!-- Load three.js -->
+<script src="https://unpkg.com/three@0.106.2/build/three.min.js"></script>
+<!-- Load scatter-gl.js -->
+<script src="https://unpkg.com/scatter-gl@0.1.0/dist/scatter-gl.min.js"></script>
```

Comment:
remove the /dist/scatter-gl.min.js

this should be taken care of by main

also if you care about stats use jsdelivr (unpkg doesnt give cdn hit counts, jsdelivr does)

Reply:
main is gonna point to the non-bundled  index.js file for npm to use, so I'll point to the build lib/scatter-gl.min.js